Emby.Naming/Video/CleanStringParser.cs

```diff
@@ -25,26 +25,35 @@ public static bool TryClean([NotNullWhen(true)] string? name, IReadOnlyList<Rege
                 return false;
             }
 
-            var len = expressions.Count;
-            for (int i = 0; i < len; i++)
+            // Iteratively apply the regexps to clean the string.
+            bool cleaned = false;
+            for (int i = 0; i < expressions.Count; i++)
             {
                 if (TryClean(name, expressions[i], out newName))
                 {
-                    return true;
+                    cleaned = true;
+                    name = newName.ToString();
                 }
             }
 
-            newName = ReadOnlySpan<char>.Empty;
-            return false;
+            newName = cleaned ? name.AsSpan() : ReadOnlySpan<char>.Empty;
+            return newName != ReadOnlySpan<char>.Empty;
```

Comment:
I think this is more intuitive
```suggestion
            return !newName.IsEmpty;
```

Reply:
I agree